### ISSUE TITLE
Core dumped

### ISSUE THREAD

**Issue by Zrufy** [Tencent/ncnn#1776]
This is the model converted from pytorch to onnx to ncnn.

`7767517
36 36
Input            input.1                  0 1 input.1
Convolution      61                       1 1 input.1 61 0=24 1=5 11=5 2=1 12=1 3=2 13=2 4=2 14=2 15=2 16=2 5=1 6=600
ReLU             62                       1 1 61 62
ConvolutionDepthWise 63                       1 1 62 63 0=24 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=216 7=24
ReLU             64                       1 1 63 64
Convolution      65                       1 1 64 65 0=128 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=3072
Pooling          66                       1 1 65 66 0=0 1=2 11=2 2=2 12=2 3=0 13=0 14=0 15=0 5=1
ConvolutionDepthWise 67                       1 1 66 67 0=128 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=1152 7=128
BatchNorm        68                       1 1 67 68 0=128
ReLU             69                       1 1 68 69
Convolution      70                       1 1 69 70 0=256 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=32768
BatchNorm        71                       1 1 70 71 0=256
ConvolutionDepthWise 72                       1 1 71 72 0=256 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=2304 7=256
ReLU             73                       1 1 72 73
Convolution      74                       1 1 73 74 0=256 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=65536
Pooling          75                       1 1 74 75 0=0 1=2 11=2 2=1 12=2 3=1 13=0 14=1 15=0 5=1
ConvolutionDepthWise 76                       1 1 75 76 0=256 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=2304 7=256
BatchNorm        77                       1 1 76 77 0=256
ReLU             78                       1 1 77 78
Convolution      79                       1 1 78 79 0=512 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=131072
BatchNorm        80                       1 1 79 80 0=512
ConvolutionDepthWise 81                       1 1 80 81 0=512 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=4608 7=512
ReLU             82                       1 1 81 82
Convolution      83                       1 1 82 83 0=512 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=262144
Pooling          84                       1 1 83 84 0=0 1=2 11=2 2=1 12=2 3=1 13=0 14=1 15=0 5=1
ConvolutionDepthWise 85                       1 1 84 85 0=512 1=2 11=2 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=2048 7=512
BatchNorm        86                       1 1 85 86 0=512
ReLU             87                       1 1 86 87
Convolution      88                       1 1 87 88 0=512 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=262144
BatchNorm        89                       1 1 88 89 0=512
Squeeze          90                       1 1 89 90 -23303=1,2
Permute          91                       1 1 90 91
Reshape          105                      1 1 91 105 0=512
InnerProduct     106                      1 1 105 106 0=128 1=1 2=65536
InnerProduct     107                      1 1 106 107 0=37 1=1 2=4736
Reshape          113                      1 1 107 113 0=-1 1=1
`

when i run the model this go to core dumped.Any help to figure out?
i use the same model dense used by https://github.com/ouyanghuiyu/chineseocr_lite.
@nihui

**Comment by Zrufy** [comment]
@nihui can you help me?

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx

**Comment by Zrufy** [comment]
i used `pnnx model.onnx inputshape=[1,224,224,3]` since the beginning but I still have this problem.

**Comment by Zrufy** [comment]
the reference model is a tensorflow model transformed into onnx via tf2onnx.
